modules/message-lane/src/lib.rs

```diff
@@ -95,6 +95,12 @@ pub trait Trait<I = DefaultInstance>: frame_system::Trait {
 	/// Identifier of relayer that deliver messages to this chain. Relayer reward is paid on the bridged chain.
 	type InboundRelayer: Parameter;
 
+	/// AccountId of the shared relayer fund account.
+	///
+	/// This account stores all the fees paid by submitters. Relayers are able to claim these
+	/// funds as at their convenience.
+	type RelayerFundAccount: Get<Self::AccountId>;
```

Comment:
is this really necessary? Can't we pass something like `hash("relayer-fund-account" ++ BridgeInstanceId)` to `AcountIdConverter`? I think it would be better to have this pallet-hardcoded than freely configurable (easier to create tooling).